src/components/navigation/document-tab-content.sass

```diff
@@ -1,54 +1,10 @@
 @import ../vars
 
-.document-tabs
-  border-color: $charcoal
-  border-width: 0 2px 2px 0
-  border-style: solid
-  &.my-work
-    border-color: $workspace-teal
-  &.class-work
-    border-color: $classwork-purple
-  &.supports
-    border-color: $support-blue
-  .tab-list
-    background-color: white
-    margin: 0
-    padding: 0
-    height: $left-tab-height
-    display: flex
-    &.my-work
-      background-color: $workspace-teal-light-7
-    &.class-work
-      background-color: $classwork-purple-light-7
-    &.supports
-      background-color: $support-blue-light-7
-    .doc-tab
-      flex-grow: 1
-      height: $left-tab-height
-      display: flex
-      justify-content: center
-      align-items: center
-      cursor: pointer
-      &.learning-logs
-        background-color: $learninglog-green-light-7
-      &:hover
-        background-color: $charcoal-light-9
-        &.my-work
-          background-color: $workspace-teal-light-5
-        &.class-work
-          background-color: $classwork-purple-light-5
-        &.supports
-          background-color: $support-blue-light-5
-        &.learning-logs
-          background-color: $learninglog-green-light-5
-      &:active, &.selected
-        background-color: $charcoal-light-8
-        font-weight: bold
-        &.my-work
-          background-color: $workspace-teal-light-3
-        &.class-work
-          background-color: $classwork-purple-light-3
-        &.supports
-          background-color: $support-blue-light-3
-        &.learning-logs
-          background-color: $learninglog-green-light-4
+.document-tab-content
+  .editable-document-content
+    top: $left-tab-height * 2
+    height: calc(100vh - (#{$header-height} + #{$workspace-content-margin} * 2 + #{$left-tab-height} * 2 + #{$tab-section-border-width} * 2))
+    position: static
+    .canvas-area
+      position: static
+      height: 100%
```

Comment:
For future reference, in situations like this where you're renaming a file and also making a bunch of changes to it, if you do the rename in one commit and then the changes in subsequent commits git can do a better job of tracking it. No reason to do anything about it for this PR but something to think about the next time it occurs.